### DISCUSSION TITLE
[Script request] NextPVR

### DISCUSSION THREAD

**Discussion by di5cord20** [community-scripts/ProxmoxVE#164] [Request script]
### Application Name

NextPVR

### Website

https://www.nextpvr.com/

### Description

NextPVR is a personal video recorder application for Microsoft Windows, Linux, Mac and Docker. NextPVR makes it easy to watch or record live TV, and provides great features like series recordings, web scheduling, iPhone/iPad client application, Kodi/Emby integration etc.

Wiki: https://github.com/sub3/NextPVR/wiki/

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by di5cord20** [comment]
Thanks for creating the script!  I was able to install but the app is extremely sluggish (using default settings).  Anyone else try the script yet and can confirm if it's a local issue on my end?

Edit: after restarting, it seems to be better.  I'll monitor and report back if I have any issues.

**Reply by MickLesk** [reply]
Yes, feedback is good, perhaps we will also increase the standard resources. Unfortunately, I could only test it visually and roughly functionally, but not properly, because I don't have the right equipment 😅
